### PR TITLE
terraform variables for FIRE_REPORT_API_URL and offline report of fir…

### PR DESCRIPTION
**Pull request by yzlucas**
…e sync interval

### CODE REVIEW

**Review by dhlevi — APPROVED** [review]

**Comment by dhlevi** [comment]
Looks like they're all there but feel free to check in with @vivid-cpreston

File: .github/workflows/android.yml [hunk 1]
```diff
@@ -118,6 +118,7 @@ jobs:
           PROJECT_VERSION: ${{ steps.version.outputs.info }}
           REPORT_A_WILDFIRE_PHONE_LINE: ${{ vars.REPORT_A_WILDFIRE_PHONE_LINE }}
           SITEMINDER_URL_PREFIX: ${{ vars.SITEMINDER_URL_PREFIX }}
+          SYNC_INTERVAL_MINUTES: ${{ var.SYNC_INTERVAL_MINUTES }}
```
Comment (vivid-cpreston) [inline .github/workflows/android.yml:121]: Typo, should be vars, not var

File: .github/workflows/ios.yml [hunk 2]
```diff
@@ -109,6 +109,7 @@ jobs:
           PROJECT_VERSION: ${{ steps.version.outputs.info }}
           REPORT_A_WILDFIRE_PHONE_LINE: ${{ vars.REPORT_A_WILDFIRE_PHONE_LINE }}
           SITEMINDER_URL_PREFIX: ${{ vars.SITEMINDER_URL_PREFIX }}
+          SYNC_INTERVAL_MINUTES: ${{ var.SYNC_INTERVAL_MINUTES }}
```
Comment (vivid-cpreston) [inline .github/workflows/ios.yml:112]: Typo, should be vars, not var

File: .github/workflows/terragrunt-deploy-manual-dev.yml [hunk 3]
```diff
@@ -131,6 +132,7 @@ jobs:
           WEBADE-OAUTH2_TOKEN_CLIENT_URL: ${{vars.WEBADE_OAUTH2_TOKEN_CLIENT_URL}}
           WEBADE-OAUTH2_TOKEN_URL: ${{vars.WEBADE_OAUTH2_TOKEN_URL}}
           WFDM_REST_URL: ${{vars.WFDM_REST_URL}}
+          FIRE_REPORT_API_URL: $${vars.FIRE_REPORT_API_URL}
```
Comment (vivid-cpreston) [inline .github/workflows/terragrunt-deploy-manual-dev.yml:135]: Only one $, should be two {

**Pull request merged**